cartography/data/jobs/analysis/aws_ec2_iaminstance.json

```diff
@@ -0,0 +1,10 @@
+{
+    "name": "EC2 Instances assume IAM roles",
+    "statements": [
+        {
+            "__comment": "Create STS_ASSUMEROLE_ALLOW realtionship from ec2 instance to the associated iaminstance iam role",
```

Comment:
Nit: How does something like "Create STS_ASSUMEROLE_ALLOW relationships from EC2 instances to the IAM roles they can assume via their iaminstanceprofiles" sound?